### PR TITLE
Reorganize gas strategy values.

### LINKED ISSUE #931: Gas price strategy details  links in `config.json`
The `deployment files > config-json.md` files contains deeply nested strategies for gas prices. The layout is hard to follow and strategies are values (not key) but presented as key in the TOC. It may be easier to follow by simply linking the stratigies to the new concept page for strategies.

### PR DESCRIPTION
**Pull request by wkande**
Closes #931 

Gas strategies are values of `options.gasPriceOracle[n].gasPriceStrategy` and not fields of the `config.json` file. The `config-json.md` file is a list of fields. The way gas strategies were listed was deeply nested into the doc and from a readability standpoint hard to follow.

I am not sure this rewrite is perfect so we can continue to look for imporovements.

### CODE REVIEW

File: docs/airnode/v0.8/concepts/gas-prices.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Gas Prices
+title: Gas Prices Strategies
```
Comment (vponline) [inline docs/airnode/v0.8/concepts/gas-prices.md]:
```suggestion
title: Gas Price Strategies
```
Should probably be like this?

**Review by vponline — APPROVED** [review]
Thanks @wkande, both the content and structural changes seem like a big improvement to me.

Perhaps @kolenic-martin  could also have a look once he's back once more, but I think this is good to go.

File: docs/airnode/v0.8/reference/deployment-files/config-json.md [hunk 2]
```diff
@@ -227,125 +227,62 @@ be repeated during Airnode's next run cycle.
...
+    Fee. Defaults: `{"value": 3.12, "unit": "gwei"}`.
+    - `priorityFee.value`: (required) - A number specifying the EIP-1559
+      priority fee value.
+    - `priorityFee.unit`: (required) - The unit of the priority fee value. It
+      can be one of the following: (wei, kwei, mwei, gwei, szabo, finney,
+      ether).
+- [constantGasPrice](../../concepts/gas-prices.md#constantgasprice)
+  - `gasPrice`: (required) - An object of the form `{"value": 0, "unit": "wei"}`
+    that configures the amount to use as gas price.
+    - gasPrice.value: (required) - A number specifying the gasPrice value.
```
Comment (dcroote) [inline docs/airnode/v0.8/reference/deployment-files/config-json.md:283]:
```suggestion
    - `gasPrice.value`: (required) - A number specifying the gasPrice value.
```

File: docs/airnode/v0.8/reference/deployment-files/config-json.md [hunk 3]
```diff
@@ -227,125 +227,62 @@ be repeated during Airnode's next run cycle.
...
+    - gasPrice.unit: (required) The unit of the gasPrice value. It can be one of
+      the following: (wei, kwei, mwei, gwei, szabo, finney, ether).
```
Comment (dcroote) [inline docs/airnode/v0.8/reference/deployment-files/config-json.md:285]:
```suggestion
    - `gasPrice.unit`: (required) The unit of the gasPrice value. It can be one
      of the following: (wei, kwei, mwei, gwei, szabo, finney, ether).
```

**Review by dcroote — COMMENTED** [review]
Two minor formatting updates, otherwise I agree, a nice restructuring👍

File: docs/airnode/v0.8/reference/deployment-files/config-json.md (hunk 3 above)
Comment (wkande) [reply docs/airnode/v0.8/reference/deployment-files/config-json.md:285]: I'll add the change to my daily updates and corrections. Thanks
